dev-php/ca-bundle/Manifest

```diff
@@ -0,0 +1,4 @@
+AUX autoload.php 268 SHA256 e4432f30b3120969792ca0af5544cbd68ca1d384a997f92f5d6424d9677a17c1 SHA512 186ccfc3ae6669722d01948c266a9c0518e1da228095a880695b7de6502c9798203019622cc0c63e30fbf7abe94aa988b7c694bcc98522e04298aa90217b8ed8 WHIRLPOOL 6632ff7476390853c5f4d93bbb58c82d6c7c8e0efdb81b93ae58dfe5e104a4e4fb248203a0ead62a23cb5e64d0723bb5ed11b3f88c874fb6e1e42423424a73e1
```

Comment:
Please regenerate those Manifests inside git repository checkout. They should not contain anything but DIST entries.